add-pkg.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash -x
+
+BUILDCONF="/usr/share/mix/builder.conf"
+
+# Auto generate builder.conf if it's not there
+if [ ! -f "$BUILDCONF" ]; then
+echo "[Builder]
+SERVER_STATE_DIR = /usr/share/mix/update
+BUNDLE_DIR = /usr/share/mix/mix-bundles
+YUM_CONF = /usr/share/mix/.yum-mix.conf
+CERT = /usr/share/mix/Swupd_Root.pem
+VERSIONS_PATH =/usr/share/mix
+RPMDIR = /usr/share/mix/rpms
+REPODIR = /usr/share/mix/local
+
+[swupd]
+BUNDLE=os-core-update
+CONTENTURL=file:///usr/share/mix/update/www
+VERSIONURL=file:///usr/share/mix/update/www
+FORMAT=1" | sudo tee "$BUILDCONF" > /dev/null
+fi
+
+MIX_DIR="/usr/share/mix/"
+OUTPUTDIR="$MIX_DIR/update/www"
+BUNDLE_DIR="$(awk -F '=' '/BUNDLE_DIR/ {print $2}' $BUILDCONF  | tr -d ' ')"
+RPM_DIR="$(awk -F '=' '/RPMDIR/ {print $2}' $BUILDCONF  | tr -d ' ')"
+REPO_DIR="$(awk -F '=' '/REPODIR/ {print $2}' $BUILDCONF  | tr -d ' ')"
+BUILD_STATE_DIR="/var/lib/mix"
+MIX_CERT="$MIX_DIR/Swupd_Root.pem"
+PRIVKEY="$MIX_DIR/private.pem"
+FORMAT="$(awk -F '=' '/FORMAT/ {print $2}' $BUILDCONF  | tr -d ' ')"
+
+pkg=$1
+bundle=$2
+
+type "mixer"
+if [ $? -ne 0]; then
```

Comment:
You need a space before the `]`, or else the shell will report a syntax error.

Reply:
🤕

Reply:
And whilst you are changing this, can you change it to

if type mixer >/dev/null ; then

to avoid a random "/usr/bin/mixer" message being output.